bin/wetzel.js

```diff
@@ -26,7 +26,9 @@ if (!defined(argv._[0]) || defined(argv.h) || defined(argv.help)) {
         '  -d,  --debug              Provide a path, and this will save out intermediate processing\n' +
         '                                artifacts useful in debugging wetzel.\n' +
         '  -w,  --suppressWarnings   Will not print out WETZEL_WARNING strings indicating identified\n' +
-        '                                conversion problems. Default: false\n';
+        '                                conversion problems. Default: false\n' +
+        '  --detailedSummary         Adds more details in the summary table. Default: false\n' +
```

Comment:
I'd rather write something more specific:
```suggestion
        '  --decribeEnums       List enum values in the description column of the summary table. Default: false\n' +
```
"Detailed summary" sounds a bit contradictory :)